### ISSUE TITLE
In show mode display the name of the sync setting in users show page

### ISSUE THREAD

**Issue by mahalakshme** [avniproject/avni-webapp#1280]
### AC:
Like shown in Edit mode of user, it needs to display the name of the sync setting instead of 'Sync Attribute 1' like in the below image:
![Image](https://github.com/avniproject/avni-webapp/assets/10369963/1cac3c28-f95f-4885-9381-36acc980c8fc)

**Comment by Joereed1008** [comment]
Please may you assign this issue to me?

**Comment by mahalakshme** [comment]
@Joereed1008 you can work on it.

**Comment by Joereed1008** [comment]
@mahalakshme Sorry I might be messing something up, new to using WSL. I've got ubuntu 22.04.3 running through WSL, have installed the distro, and have used `make deps` and `make start-with-staging` successfully. After navigating to localhost:6010, I clicked cognito, and signed in with admin@jnpct and password. 

I seem to be unable to recreate this issue at this point, with example user screen as follows:

![image](https://github.com/avniproject/avni-webapp/assets/120391655/ef80ba65-d710-4e1c-997e-a4ad8f214ccc)

What am I missing?
